wsgi/odyssy/basic/templates/basic/index.html

```diff
@@ -36,17 +36,17 @@
             <!-- Announcements -->
             <div class="col-sm-4" id="announcements">
                 <div class='announcement-content '>
-                    <div class="widgets-title"><h2>Announcements</h2></div>
+                    <div class="widgets-title"><h2><a href="/announcement/">Announcements</a></h2></div>
```

Comment:
https://github.com/iiitv/Odyssy/blob/master/wsgi/odyssy/announcement/urls.py
here it is mentioned `name='index-view'` you can use that.